solaris/solaris11/SPECS/template_agent_v4.2.0.json

```diff
@@ -671,6 +671,14 @@
         "type": "file",
         "user": "ossec"
     },
+    "/var/ossec/logs/wazuh": {
+        "class": "static",
+        "group": "ossec",
+        "mode": "0750",
+        "prot": "drwxr-x---",
+        "type": "directory",
+        "user": "ossec"
+    },
     "/var/ossec/logs/ossec": {
```

Comment:
Have we to remove `/var/ossec/logs/ossec`? and `/var/ossec` still being `ossec?`.

Reply:
Yhis changes follow this issue wazuh/wazuh#6885, for the moment the installation directory will still be `/var/ossec`